### PR TITLE
[denonmarantz] Made constants and handler packages internal

### PR DESCRIPTION
**Pull request by VelinYordanov**
…inding

See eclipse/smarthome#4330

Signed-off-by: VelinYordanov <velin.iordanov@gmail.com>

### CODE REVIEW

File: addons/binding/org.openhab.binding.denonmarantz/src/main/java/org/openhab/binding/denonmarantz/internal/handler/DenonMarantzHandler.java [hunk 1]
```diff
@@ -44,7 +44,7 @@
 import org.eclipse.smarthome.core.types.Command;
 import org.eclipse.smarthome.core.types.RefreshType;
 import org.eclipse.smarthome.core.types.State;
-import org.openhab.binding.denonmarantz.DenonMarantzBindingConstants;
+import org.openhab.binding.denonmarantz.internal.DenonMarantzBindingConstants;
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.denonmarantz/src/main/java/org/openhab/binding/denonmarantz/internal/handler/DenonMarantzHandler.java]: Can we maybe reafactor this class to use `DenonMarantzBindingConstants` as static import only?

File: addons/binding/org.openhab.binding.denonmarantz/src/main/java/org/openhab/binding/denonmarantz/internal/discovery/DenonMarantzDiscoveryParticipant.java [hunk 2]
```diff
@@ -25,7 +25,7 @@
 import org.eclipse.smarthome.core.thing.Thing;
 import org.eclipse.smarthome.core.thing.ThingTypeUID;
 import org.eclipse.smarthome.core.thing.ThingUID;
-import org.openhab.binding.denonmarantz.DenonMarantzBindingConstants;
+import org.openhab.binding.denonmarantz.internal.DenonMarantzBindingConstants;
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.denonmarantz/src/main/java/org/openhab/binding/denonmarantz/internal/discovery/DenonMarantzDiscoveryParticipant.java]: Can we maybe reafactor this class to use `DenonMarantzBindingConstants` as static import only?

File: addons/binding/org.openhab.binding.denonmarantz/src/main/java/org/openhab/binding/denonmarantz/internal/connector/DenonMarantzConnector.java [hunk 3]
```diff
@@ -19,7 +19,7 @@
 import org.eclipse.smarthome.core.library.types.StringType;
 import org.eclipse.smarthome.core.types.Command;
 import org.eclipse.smarthome.core.types.RefreshType;
-import org.openhab.binding.denonmarantz.DenonMarantzBindingConstants;
+import org.openhab.binding.denonmarantz.internal.DenonMarantzBindingConstants;
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.denonmarantz/src/main/java/org/openhab/binding/denonmarantz/internal/connector/DenonMarantzConnector.java]: Can we maybe reafactor this class to use `DenonMarantzBindingConstants` as static import only?

File: addons/binding/org.openhab.binding.denonmarantz/src/main/java/org/openhab/binding/denonmarantz/internal/xml/adapters/VolumeAdapter.java [hunk 4]
```diff
@@ -12,7 +12,7 @@
 
 import javax.xml.bind.annotation.adapters.XmlAdapter;
 
-import org.openhab.binding.denonmarantz.DenonMarantzBindingConstants;
+import org.openhab.binding.denonmarantz.internal.DenonMarantzBindingConstants;
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.denonmarantz/src/main/java/org/openhab/binding/denonmarantz/internal/xml/adapters/VolumeAdapter.java]: Can we maybe reafactor this class to use `DenonMarantzBindingConstants` as static import only?

**Comment by wborn** [comment]
The minor comments have been addressed @cweitkamp!

**Pull request merged**